cli/args/flags.rs

```diff
@@ -3810,6 +3821,7 @@ fn reload_arg_validate(urlstr: &str) -> Result<String, String> {
 fn watch_arg_parse(matches: &mut ArgMatches) -> Option<WatchFlags> {
   if matches.get_flag("watch") {
     Some(WatchFlags {
+      hot_reload: matches.get_flag("hot-reload"),
```

Comment:
Refactor this to not require `--watch` and `--hot` - only one of them should be required:
a) `--watch` - no HMR
b) `--hot` - `--watch` + `hot_reload: true`